src/VisualStudio/CSharp/Impl/Options/Formatting/StyleViewModel.cs

```diff
@@ -1006,6 +1006,45 @@ void Method()
 //]
     }
 }
+";
+
+        private const string s_preferExpressionBodyForLocalFunctions = @"
+using System;
+
+//[
```

Comment:
I believe the `//[` and `//]` markers could be scoped more narrowly, to be consistent with other examples in this file. They could just be around the local function.
That said, I don't exactly know what that affects. @CyrusNajmabadi do you know?

Reply:
Only the part shown between these markers in shown in the code style preview window. If I make this change, it would become like this:
![image](https://user-images.githubusercontent.com/30838294/50166930-47b19780-02e8-11e9-9df5-3d7f7ca70928.png)

Since you can't see it's a local function anymore, I think we shouldn't make this change.